integ-test/src/test/resources/null_missing.json

```diff
@@ -0,0 +1,18 @@
+{"index":{}}
```

Comment:
you don't want to give these index numbers?

Reply:
They assigned automatically if not given.